tests/MassTransit.HangfireIntegration.Tests/MassTransit.HangfireIntegration.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Hangfire.AspNetCore" Version="1.7.28"/>
```

Comment:
Have they not built a lighter weight Extensions.Hosting or Extensions.DI package yet?

Reply:
nope, I do not like their package separation either